WcaOnRails/app/models/user.rb

```diff
@@ -331,6 +331,18 @@ def avatar_requires_wca_id
     end
   end
 
+  def send_on_create_confirmation_instructions
+    NewRegistrationMailer.send_registration_mail(self).deliver_now
+  end
+
+  def send_reconfirmation_instructions
```

Comment:
Can you add a comment here about how this was copied from https://github.com/plataformatec/devise/blob/master/lib/devise/models/confirmable.rb#L124?

Reply:
Sure, will do!